packages/core/src/api/nodeConversions/nodeToBlock.ts

```diff
@@ -65,10 +65,8 @@ export function contentNodeToTableContent<
 
       return {
         type: "tableCell",
-        content: contentNodeToInlineContent(
-          cellNode.firstChild!,
-          inlineContentSchema,
-          styleSchema
+        content: cellNode.content.content.flatMap((child) =>
```

Comment:
are we sure this doesn't load any information when we load the blocknote json in the editor again? i.e.: should there be a `join("\n")` for example, or similar?

Reply:
I'll take a look at this again, but, yea there definitely is some sort of coalescing that can happen here.